backend/workflow_manager/endpoint/exceptions.py

```diff
@@ -69,3 +71,43 @@ class BigQueryTableNotFound(APIException):
         "Please enter correct correct bigquery table in the form "
         "{table}.{schema}.{database}."
     )
+
+
+class UnstractDBException(APIException):
+    status_code = 400
+    default_detail = "Error writing to database"
+
+    def __init__(self, code: Any = status_code, detail: str = default_detail) -> None:
+        super().__init__(detail=detail, code=code)
+
+
+class InvalidSyntaxException(UnstractDBException):
+    def __init__(self, code: Any, detail: Any) -> None:
+        default_detail = "Syntax incorrect. "
```

Comment:
Just showing `Syntax incorrect. <Some PG error>` might not be as helpful - it'll be good if we can show `Error writing to <DB name>. Syntax incorrect, <PG error>`

Reply:
@chandrasekharan-zipstack getting db_name is not possible as its being initialized while creating UnstractDB class (for eg Postgres). I have changed the name to `table_name' and updated the message. Please check